src/components/ScenarioParameters/components/EditModeButton.js

```diff
@@ -3,29 +3,22 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Grid, Button } from '@material-ui/core';
-import { useTranslation } from 'react-i18next';
+import { Grid, IconButton } from '@material-ui/core';
 import PlayCircleOutlineIcon from '@material-ui/icons/PlayCircleOutline';
+import BackspaceIcon from '@material-ui/icons/Backspace';
 
 const EditModeButton = ({ classes, handleClickOnDiscardChange, handleClickOnUpdateAndLaunchScenario }) => {
-  const { t } = useTranslation();
   return (
     <Grid container spacing={1}>
       <Grid item>
-        <Button data-cy="discard-button" color="primary" onClick={handleClickOnDiscardChange}>
-          {t('commoncomponents.button.scenario.parameters.discard', 'Discard Modifications')}
-        </Button>
+        <IconButton data-cy="discard-button" color="primary" onClick={handleClickOnDiscardChange}>
+          <BackspaceIcon />
+        </IconButton>
       </Grid>
       <Grid item>
-        <Button
-          data-cy="update-and-launch-scenario"
-          startIcon={<PlayCircleOutlineIcon />}
-          variant="contained"
-          color="primary"
-          onClick={handleClickOnUpdateAndLaunchScenario}
-        >
-          {t('commoncomponents.button.scenario.parameters.update.launch', 'Update And Launch Scenario')}
-        </Button>
+        <IconButton data-cy="update-and-launch-scenario" color="primary" onClick={handleClickOnUpdateAndLaunchScenario}>
+          <PlayCircleOutlineIcon />
+        </IconButton>
```

Comment:
Do you think we could keep some tooltips on these icon buttons (edit, launch, discard, update & launch) ? It may be less disruptive for previous users, and provide information for new users

Reply:
Sure